### PR TITLE
added enum ProductFilter

### PR DESCRIPTION
**Pull request by MyroslavaVynokur**
added ResultsFilter implementation

### CODE REVIEW

File: src/main/java/com/softserveinc/ita/rozetka/components/ResultsFilter.java [hunk 1]
```diff
@@ -1,5 +1,23 @@
...
+import utils.ProductFilter;
+
+import java.util.List;
+
+import static com.codeborne.selenide.Selenide.$x;
+
 public class ResultsFilter extends Header {
-    // implement
+
+    public SearchResultsPage toggleFilter(ProductFilter type) {
```
Comment (toleksyn) [inline src/main/java/com/softserveinc/ita/rozetka/components/ResultsFilter.java:12]: rename, filter()

File: src/main/java/com/softserveinc/ita/rozetka/components/ResultsFilter.java [hunk 2]
```diff
@@ -1,5 +1,23 @@
...
+        $x(String.format("//a[@data-id = '%s']", type.getFilterXpath())).click();
+        return new SearchResultsPage();
+    }
+
+    public SearchResultsPage toggleFilter(List<ProductFilter> filters) {
```
Comment (toleksyn) [inline src/main/java/com/softserveinc/ita/rozetka/components/ResultsFilter.java:17]: same

File: src/main/java/com/softserveinc/ita/rozetka/components/ResultsFilter.java [hunk 3]
```diff
@@ -1,5 +1,23 @@
...
+        for (ProductFilter filter : filters) {
+            $x(String.format("//a[@data-id = '%s']", filter.getFilterXpath())).click();
+        }
```
Comment (toleksyn) [inline src/main/java/com/softserveinc/ita/rozetka/components/ResultsFilter.java:20]: use foreach

**Review by toleksyn — CHANGES_REQUESTED** [review]

File: src/main/java/utils/ProductFilter.java [hunk 4]
```diff
@@ -0,0 +1,20 @@
+package utils;
```
Comment (toleksyn) [inline src/main/java/utils/ProductFilter.java:1]: add com.softserve... package
this is not util, this is data

File: src/main/java/com/softserveinc/ita/rozetka/components/ResultsFilter.java (hunk 3 above)
Comment (MyroslavaVynokur) [reply src/main/java/com/softserveinc/ita/rozetka/components/ResultsFilter.java:20]: foreach is used in method toggleFilter. Could you please clarify what should be corrected?

**Review by toleksyn — CHANGES_REQUESTED** [review]

File: src/main/java/com/softserveinc/ita/rozetka/components/ResultsFilter.java (hunk 3 above)
Comment (toleksyn) [reply src/main/java/com/softserveinc/ita/rozetka/components/ResultsFilter.java:20]: I meant the lambda one

**Review by toleksyn — CHANGES_REQUESTED** [review]

**Pull request merged**